rhel_aws/README.md

```diff
@@ -1,19 +1,11 @@
 # RHEL 8 Workshop - AWS Version
 
-![ansible](img/Ansible-Tower-Logotype-Large-RGB-FullGrey-300x124.png)
+![rhel 8](img/Logo-Red_Hat-Enterprise_Linux_8-B-Standard-RGB.png)
 
 `rhel8_aws` is a playbook to provision RHEL 8 in AWS. This playbook uses Ansible to provision AWS infrastructure and nodes.
 
 These modules all require that you have AWS API keys available to use to provision AWS resources. You also need to have IAM permissions set to allow you to create resources within AWS. There are several methods for setting up your AWS environment, on you local machine.
 
-Fill out `env.sh` & Export the AWS API Keys
-
-First, copy env.sh_example to env.sh, and then fill in your API keys.  Once that is complete, source the script, to export your AWS environment variables.
-
-```
-source env.sh
-```
-
 This repo also requires that you have Ansible installed on your local machine. For the most upto date methods of installing Ansible for your operating system [check here](http://docs.ansible.com/ansible/intro_installation.html).
```

Comment:
Why not say, "To view Ansible installation methods according to your operating system"